tests/cross/src/test/scala/tests/hover/HoverScala3TypeSuite.scala

```diff
@@ -63,7 +63,7 @@ class HoverScala3TypeSuite extends BaseHoverSuite {
        |   case Red, Green, Blue
        |
        |""".stripMargin,
-    """|enum Color: enums2.SimpleEnum
```

Comment:
`enum Color: Color` seems less informative than the previous version. I would opt to leave this one as is.

Reply:
My reasoning for this was that for example if we have `case class Foo()` and hover on `Foo()` we'll get `Foo: Foo`, which this sort of aligns with. Morever, `enums2.SimpleEnum` is actually incorrectly here, that's not the type. It could be `enums2.SimpleEnum.Color`, which would be fully qualified and correct, but very verbose. So mainly I just think` enums2.SimpleEnum` is incorrect, since it's showing you the object, but not the actual enum type. wdyt?

Reply:
This is actually how it's handled in other cases like traits, so I guess it's better to do it this way :+1:

Reply:
It seems that the `HoverTermSuite` is failing though so there might be some more cases to fix.

Reply:
Yea just noticed that and actually is shows a nice example

```scala
FileVisi@@tResult.CONTINUE
```
Used to return `FileVisitResult: java.nio.file` before, which I think is confusing, so I like it better now that it just shows `enum FileVisitResult: FileVisitResult`.

Reply:
I kind of liked that :sweat_smile:

Reply:
haha I don't have strong feelings on this, and this was sort of just me poking around, so by all means push back.

For me I guess I'm just trying to understand _what_ should actually be shown here. Like what is the desired type to show for `FileVisitResult` for example. Like it is an enum, so is showing the package name for example correct as the type? My gut says no, which is why I made the change, but I might totally be missing something.

Reply:
I think actually there is no need to show the type in this case, but we can change that separately at some point alongside traits and classes.